File: src/cryptonite-app/src/app/models/base/paginatedSearch.ts
import { SortDirection } from './sortDirection';

export interface PaginatedSearch {
  pageIndex: number;
  pageSize: number;
  sortColumn: string;
  sortDirection: SortDirection;
  filterType: any;
  term: string;
  id?: string;
  relation?: string;
  isFormOpen?: boolean;
  underEditId?: string;
}

File: src/Cryptonite.Infrastructure/Common/PaginatedSearch.cs
using Cryptonite.Infrastructure.Data.Common;

namespace Cryptonite.Infrastructure.Common
{
    public class PaginatedSearch
    {
        public string Term { get; set; }
        public int PageIndex { get; set; }
        public int PageSize { get; set; }
        public string SortColumn { get; set; }
        public SortDirection SortDirection { get; set; }
    }
}